### ISSUE TITLE
Using labels for other purposes (extracting labels from detections)

### ISSUE THREAD

**Issue by prp-e** [ultralytics/yolov5#5925]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

I have some ideas in mind and I found yolov5 the easiest and most accessible tool for making a good model in a short time. But I also am confused on how I can extract labels from the results.
This is how I get the model from my custom model:

```
model = torch.hub.load('ultralytics/yolov5', 'custom', path='yolov5/runs/train/exp8/weights/last.pt', force_reload=True)
```

and detection/prediction like this:

```
result = model(image)
```

Now, consider I have three different labels A, B and C. I want to do something like this (pseudo-code, of course):

```
if label == A
    goto http://example.com
if label == B
    goto http://example2.com
```

Or, just store labels in a file for doing the next procedures and stuff. 
I'll be very glad to have your helps and guidance on this project/idea. 

Thanks!

**Comment by glenn-jocher** [comment]
@prp-e see [PyTorch Hub tutorial](https://docs.ultralytics.com/yolov5/tutorials/pytorch_hub_model_loading) for handling results:

```python
import torch

# Model
model = torch.hub.load('ultralytics/yolov5', 'yolov5s')

# Image
img = 'https://ultralytics.com/images/zidane.jpg'

# Inference
results = model(img)

results.pandas().xyxy[0]
#      xmin    ymin    xmax   ymax  confidence  class    name
# 0  749.50   43.50  1148.0  704.5    0.874023      0  person
# 1  433.50  433.50   517.5  714.5    0.687988     27     tie
# 2  114.75  195.75  1095.0  708.0    0.624512      0  person
# 3  986.00  304.00  1028.0  420.0    0.286865     27     tie
```
